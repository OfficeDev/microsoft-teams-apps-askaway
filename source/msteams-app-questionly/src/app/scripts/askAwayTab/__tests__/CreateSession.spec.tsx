import { Flex } from '@fluentui/react-northstar';
import { configure, shallow } from 'enzyme';
import enzymeAdapterReact16 from 'enzyme-adapter-react-16';
import enzymeToJson from 'enzyme-to-json';
import * as React from 'react';
import { CreateSession } from '../popups/CreateSession';

configure({ adapter: new enzymeAdapterReact16() });

describe('Create session', () => {
    it('should match the snapshot', () => {
        const wrapper = shallow(<CreateSession />);
        expect(enzymeToJson(wrapper)).toMatchSnapshot();
    });

    it.skip('should render TabContent', () => {
        const component = shallow(<CreateSession />);

        expect(component.find(Flex)).toHaveLength(2);
    });
});
