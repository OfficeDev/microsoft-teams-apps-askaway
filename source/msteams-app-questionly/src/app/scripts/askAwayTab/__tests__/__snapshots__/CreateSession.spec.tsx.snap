// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create session should match the snapshot 1`] = `
<Provider
  theme={Object {}}
>
  <div
    style={
      Object {
        "padding": "1rem 2rem",
      }
    }
  >
    <Flex
      column={true}
    >
      <Form
        as="form"
        className="sidepanel-form"
        onSubmit={[Function]}
        styles={
          Object {
            "display": "flex",
            "flexDirection": "column",
          }
        }
      >
        <div
          className="form-grid"
        >
          <Text
            as="span"
            content="Title Name*"
            size="small"
          />
          <Input
            accessibility={[Function]}
            as="div"
            errorIndicator={<ExclamationCircleIcon />}
            fluid={true}
            iconPosition="end"
            label=""
            onChange={[Function]}
            onKeyUp={[Function]}
            placeholder="Type a name"
            styles={
              Object {
                "color": "#c8c6c4",
              }
            }
            successIndicator={<PresenceAvailableIcon />}
            type="text"
            wrapper={Object {}}
          />
        </div>
        <div
          className="form-grid"
        >
          <Text
            as="span"
            content="Description*"
            size="small"
          />
          <TextArea
            accessibility={[Function]}
            as="textarea"
            fluid={true}
            onChange={[Function]}
            onKeyUp={[Function]}
            placeholder="Type a description"
            styles={
              Object {
                "marginTop": "0.25rem",
              }
            }
          />
        </div>
        <div
          className="form-grid"
        >
          <Flex>
            <FlexItem
              push={true}
            >
              <Button
                accessibility={[Function]}
                as="button"
                className="btn-create-session"
                primary={true}
                size="small"
                type="submit"
              >
                <ButtonContent
                  as="span"
                >
                  Create
                </ButtonContent>
              </Button>
            </FlexItem>
          </Flex>
        </div>
      </Form>
    </Flex>
  </div>
</Provider>
`;
